packages/model-viewer/src/three-components/EnvironmentScene.ts

```diff
@@ -29,16 +29,16 @@ export default class EnvironmentScene extends Scene {
     const boxMaterial = new MeshStandardMaterial({metalness: 0});
 
     const mainLight = new PointLight(0xffffff, 500.0, 28, 2);
-    mainLight.position.set(0.418, 16.199, 0.300);
+    mainLight.position.set(0.5, 18.0, 0.5);
```

Comment:
I'm trying to figure out why the floor looks darker in these reflections than it did. My only guess is it's because the main light moved up by 2? Can you take a look at these height dimensions (of the room as well) and see if we can get the reflection to match the original a bit closer? None of this should have any noticeable effect on the lighting of your model.

Reply:
Hi, by adjust the room scale and tweak the point light height, I brought the reflection image of the room closer to the current version. Please take a look.

Reply:
Thanks! I don't suppose I could trouble you to post updated renders here? The ones you did before were very helpful.

Reply:
Definitely. Here's the comparison. Left is the original, Right is the new environment.

![modeViewerCompare_20201215](https://user-images.githubusercontent.com/75750117/102289954-95782f00-3f05-11eb-8ba1-08125e0646ee.png)

Reply:
Thanks, that looks great! I'm going to hold off on merging this for a bit as there are some other's who want to take a look at it. I'll make sure it gets in for the next release, which is looking early January now.